.github/workflows/master.yml

```diff
@@ -183,6 +183,20 @@ jobs:
           source activate test-environment
           ./dev/run-python-flavor-tests.sh;
 
+  mlflow:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: Install mlflow
+        run: |
+          pip install -e .
+      - name: Verify mlflow can be imported without errors
+        run: |
+          python -c "import mlflow"
```

Comment:
@tomasatdatabricks Added a new job to verify `mlflow` can be installed without errors.

Reply:
Thanks!